packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -178,7 +174,7 @@ const Category = props => {
                 classes={classes}
                 data={loading ? null : data}
                 pageControl={pageControl}
-                sortControl={sortControl}
+                sortProps={sortProps}
```

Comment:
Seems like surface area for an inconsistency / bug to creep in when both the `CategoryContent` and the `pageControl` passed to the `CategoryContent` need sort data.